src/i18n/locales/ja.json

```
@@ -10,6 +10,10 @@
     "loginOrOutPrompt": "リンクをクリックしてサイトににログインまたはログアウトして閉じてください。",
     "notice": "注意",
     "secretPrompt": "シークレットトークンは端末に残り、他の人と共有されません。"
```

```suggestion
    "secretPrompt": "シークレットトークンは端末に残り、他の人と共有されません。",
```

The `,` is required.